src/tests/const.test.js

```diff
@@ -0,0 +1,78 @@
+import { createHeadlessForm } from '../createHeadlessForm';
+
+it('Should work for number', () => {
+  const { handleValidation } = createHeadlessForm(
+    {
+      properties: {
+        ten_only: { type: 'number', const: 10 },
+      },
+    },
+    { strictInputType: false }
+  );
+  expect(handleValidation({}).formErrors).toEqual(undefined);
+  expect(handleValidation({ ten_only: 1 }).formErrors).toEqual({
+    ten_only: 'The only accepted value is 10.',
+  });
+  expect(handleValidation({ ten_only: 10 }).formErrors).toEqual(undefined);
```

Comment:
todo: It's missing to test the `ten_only: null` to cover the "bug" `@BUG RMT-518` (check other comments in the `createHeadlessForm.test.js`)

```js
  // null is also considered valid until we fix @BUG RMT-518
  // Expectation: To fail with error "The only accepted value is 10."
  expect(handleValidation({ ten_only: null }).formErrors).toBeUndefined();
```

Reply:
Added!